scripts/scripts.js

```diff
@@ -148,9 +148,13 @@ const eagerLoad = (img) => {
 };
 
 (async function loadLCPImage() {
-  const marquee = document.querySelector('.marquee.split');
+  const marquee = document.querySelector('.marquee');
   if (marquee) {
-    marquee.querySelectorAll('img').forEach(eagerLoad);
+    if (marquee.classList.contains('split')) {
+      marquee.querySelectorAll('img').forEach(eagerLoad);
+    } else {
+      eagerLoad(marquee.querySelector('img'));
+    }
   } else {
     eagerLoad(document.querySelector('img'));
   }
```

Comment:
We try to keep a flat structure and avoid nested ifs. What about:

```
const marquee = document.querySelector('.marquee');
if (!marquee) {
  eagerLoad(document.querySelector('img'));
  return;
}

if (marquee.classList.contains('split')) {
  marquee.querySelectorAll('img').forEach(eagerLoad);
} else {
  eagerLoad(marquee.querySelector('img'));
}
```

Reply:
Agree. We could even drop the final `else` to accent the default behaviour
```
if (marquee.classList.contains('split')) {
  marquee.querySelectorAll('img').forEach(eagerLoad);
  return; // Alternatively, return the statement above to get to a one-liner
}

eagerLoad(marquee.querySelector('img'));
```